Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -402,7 +403,16 @@ private sealed class ReloadModulesQueueItem : IAnalyzable {
                 TraceMessage($"Only returning {filterKind.Value} items");
                 members = members.Where(m => m.kind == filterKind.Value);
             }
-
+            if (members != null && members.Count() > 0)
+            {
+                //provide Pythia recommendations based on the default completion list
+                var recommendations = PythiaService.Instance.GetRecommentations(members, tree, @params, 5);
```

Comment:
I think it may be worth try-catch and log exceptions. This way Pythia problems won't affect general functionality

Reply:
Also, tests should be there